Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/MobilityCommonBridge.java

```diff
@@ -5226,9 +5226,17 @@ public String saveAudioFile(String source) throws IOException {
     }
 
     @JavascriptInterface
-    public void uploadFile() {
-        if(mediaPlayer == null) mediaPlayer = new MediaPlayer(bridgeComponents);
-        mediaPlayer.uploadFile();
+    public void uploadFile(String ratio) {
+        try{
+            if(mediaPlayer == null) mediaPlayer = new MediaPlayer(bridgeComponents);
+            JSONObject payload = new JSONObject(ratio);
+            int imageAspectHeight = payload.optInt("imageAspectHeight", 0);
+            int imageAspectWidth = payload.optInt("imageAspectWidth", 0);
+            boolean showAccordingToAspectRatio = payload.optBoolean("showAccordingToAspectRatio", false);
+            mediaPlayer.uploadFile(imageAspectHeight, imageAspectWidth, showAccordingToAspectRatio);
```

Comment:
is it okay to have default height and width 0?

Reply:
yea because where i am making their height and width as 0 , i am passing showAccordingToAspectRatio as false and at the end i have put the check on showAccordingToAspectRatio if it true then only there is a use of height and width